src/displayapp/screens/Tab.cpp

```diff
@@ -19,7 +16,7 @@ extern lv_font_t jetbrains_mono_bold_20;
 //  screen->OnObjectEvent(obj, event);
 //}
 
-Tab::Tab(DisplayApp* app, Pinetime::Components::Gfx &gfx) : Screen(app, gfx) {
+Tab::Tab(DisplayApp* app, Pinetime::Components::Gfx &gfx) : Screen(app), gfx(gfx) {
```

Comment:
To be honest, this is a weird autoformatting. The duplicated `gfx` doesn't seem to be adding any clarity.

Reply:
No it's correct. The screen constructor only takes one argument, which is the DisplayApp pointer `app`. `gfx` is the member reference-variable, that is initialized with the parameter which is also called `gfx`.

Reply:
As I wrote above, I think we can remove this Tab class and forget about it :)